SwanCustomEnvironments/swancustomenvironments/scripts/builders/accpy.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Checks if the provided Acc-Py version is valid
+if [ -n "$BUILDER_VERSION" ] && [ ! -e "$ACCPY_PATH/base/$BUILDER_VERSION" ]; then
```

Comment:
If `$BUILDER_VERSION` is not defined we will continue running the script, we need to protect for that case and error out too.

Reply:
Actually that initial condition is redundat. The `$BUILDER_VERSION ` is mandatory and the script stops at the beginning of `makenv.sh`